container_files/mldb_nginx_site.conf

```diff
@@ -87,12 +87,20 @@ server {
             return 301 $scheme://$http_host{{HTTP_BASE_URL}}/doc/;
         }
 
-        location {{HTTP_BASE_URL}}/doc/ {
-            # send documentation render requests to mldb (.md.html)
-            rewrite ^{{HTTP_BASE_URL}}(/?.*) $1 break;
-            try_files $uri $uri/ @mldb;
+        location = {{HTTP_BASE_URL}}/resources {
```

Comment:
Indeed, I added it because I was manually testing redirect but I did not see a link to this URL.